assets/src/components/LayoutSelector/LayoutSelector.js

```diff
@@ -19,12 +19,17 @@ export class LayoutSelector extends Component {
     return <div className='LayoutSelector'>
       {
         this.props.options.map((layoutOption, i) => {
+
+          if ( 'number' == typeof layoutOption.value ) {
```

Comment:
Hey @dpivo I think we should rather avoid doing type-checking and/or casting in the LayoutSelector (as we did in the past!) could you just pass the values as strings as in the Covers block: https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/develop/assets/src/blocks/Covers/Covers.js#L194 ?

The value is already being casted to number in the Gallery block so I guess it should work right away:
https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/develop/assets/src/blocks/Gallery/GalleryBlock.js#L160

Thanks!

Reply:
Hey @pablocubico I did it like this because even though the 'gallery_block_style' is being cast to a number, when it is passed to the LayoutSelector it for some reason get's cast to a string. but this is only when changing the style, on editing of the block if a style has previously been selected, it pulls through as a number when the block is opened for editing.

Reply:
@pablocubico @dpivo Tested locally and seems okay

Reply:
@dpivo I hear ya, maybe I'm being too defensive, let's go with your solution and make the LayoutSelector smarter then, it's not inside a loop running 10 million times anyway. Let's just test this doesn't break the selector on any of the other blocks where it's used.